### ISSUE TITLE
mono-helloworld doesn't build (needed for test cases)

### ISSUE THREAD

**Issue by ajlennon** [DynamicDevices/meta-mono#24]

**Comment by ajlennon** [comment]
pkg-config isn't finding mono support for some reason 

```
| checking for x86_64-poky-linux-strip... x86_64-poky-linux-strip
| checking for a thread-safe mkdir -p... /data_drive/git/monotest/poky/build/tmp/hosttools/mkdir -p
| checking for gawk... gawk
| checking whether make sets $(MAKE)... yes
| checking whether make supports nested variables... yes
| checking for pkg-config... /data_drive/git/monotest/poky/build/tmp/work/core2-64-poky-linux/mono-helloworld/1.2-r0/recipe-sysroot-native/usr/bin/pkg-config
| checking for csc... no
| checking for mcs... no
| checking for mono... no
| configure: error: No C# compiler found
| WARNING: exit code 1 from a shell command.
```

**Comment by ajlennon** [comment]
FIxed in #bb6c7d887bd20bdd7c16ba9aaf37df2b24055f01
